docs/assets/js/charts.js

```diff
@@ -382,14 +402,51 @@ function createHistoryChart(canvas, actionBar)
                 actionBar.prepend(buttons);
             }
 
+            const tooltipTitleCallback =
+                function(tooltipItem, data)
+                {
+                    // In the history chart, tooltips are rendered for individual data points.
+                    // Hence, the tooltipItem array contains exactly one element
+                    const dataPoint = data.datasets[tooltipItem[0].datasetIndex].data[tooltipItem[0].index];
+
+                    const date = dataPoint.x;
+                    const dateRange = dataPoint.dateRange;
+
+                    const suffix = (dataPoint.infoText === undefined)
+                        ? ''
+                        : ' (' + dataPoint.infoText + ')';
+
+                    if (dateRange === undefined)
+                        return moment(date).utc().format('D MMM YYYY') + suffix;
```

Comment:
Would it make sense to define the tooltip format as some kind of global constant? We replicate it 5 times in this PR.

Reply:
I also thought about this. However, the interface of Moments.js doesn’t look like this could be extracted.

I would have expected Moments.js to work like this:

```
moment.utc().format('D MMM YYYY')(1234567890);
```

This would have enabled us to extract the formatting function like this:

```
const formatTime = moment.utc().format('D MMM YYYY');
formatTime(1234567890);
```

However, as you can see, the interface isn’t like this. Yes, we could move the format into a global configuration, but then there’s still the UTC part that’s replicated everywhere.

Reply:
But now thinking of this, let me have a quick look whether there is another interface that would do the trick.

Reply:
We could also simply write a wrapper function:

```js
function formatTime(time)
{
    return moment(time).utc().format('D MMM YYYY');
}
```

Would you prefer it in this way?